app/Http/Controllers/PostController.php

```diff
@@ -24,9 +24,7 @@ public function store(Request $request): RedirectResponse
         $post = new Post;
         $post->title = $request->input('title');
         $post->body = $request->input('body');
-
```

Comment:
Por temas de legibilidad del código, es recomendable dejar líneas en blanco entre las instrucciones, al menos en declaraciones que no sean secuencias de operaciones equivalentes.

No hay una regla exacta para esto, es una cuestión de lo que se ve mejor en su contexto.

Más respecto a este punto puedes revisarlo en la guía que adoptamos como referencia para nuestro proyecto:
https://spatie.be/guidelines/laravel-php#content-whitespace

Fíjate también en este código, que forma parte parte del core de laravel, cómo usan las líneas en blanco.
https://github.com/laravel/framework/blob/9.x/src/Illuminate/Console/Application.php

Para el caso específico de este controlador, se ve un poco mejor el código organizado de la siguiente forma y no todo "amuñuñado" 🤣
```php
public function store(Request $request): RedirectResponse
{
    $request->validate([
        'title' => ['required', 'min:1', 'max:30'],
        'body' => ['required', 'max:250'],
    ]);

    $post = new Post;
    $post->title = $request->input('title');
    $post->body = $request->input('body');

    $post->save();

    session()->flash('status', 'Post Created!!');

    return to_route('posts.create');
}
```

Reply:
ok.